blackbox/docs/general/ddl/analyzers.rst

```diff
@@ -415,12 +415,12 @@ token_chars
 
 .. _keyword-tokenizer:
 
-Keywork Tokenizer
+Keyword tokenizer
 -----------------
 
 ``type='keyword'``
 
-The ``keyworkd`` tokenizer emits the entire input as a single token.
+The ``keyword`` tokenizer emits the entire input as a single token.
```

Comment:
revert this. not related to PR. though feel free to open a new PR to fix this typo!

Reply:
Done. I will include this in the fixups PR.